simulation/traffic_simulator/src/vehicle_model/sim_model_ideal_steer_acc.cpp

```diff
@@ -1,10 +1,11 @@
+
```

Comment:
Is this blank line intended?
If not intended, please remove.

Reply:
```suggestion
```